py/tests/algorithms/anomaly_likelihood_test.py

```diff
@@ -329,8 +329,8 @@ def testSerialization(self):
 
 class AnomalyLikelihoodAlgorithmTest(TestCaseBase):
   """Tests the low-level algorithm functions"""
-  #seed for function _generateSampleData()
-  #zero means really random and not deterministic 
+  #Seed for function _generateSampleData()
+  #Generator is really random and not deterministic when 'None' is given  
```

Comment:
nit, please add a note that "unlike htm::Random, the numpy.random may not give deterministic results on all platforms"

Reply:
Actually, `numpy.random` uses the Mersenne Twister.  The numpy docs claim it is deterministic https://docs.scipy.org/doc/numpy/reference/generated/numpy.random.RandomState.html#numpy.random.RandomState